lib/params/ColorMap.cpp

```diff
@@ -425,6 +425,47 @@ ColorMap::Color ColorMap::color(float value) const
     return colorNormalized(nv);
 }
 
+static void lab2lch(const float lab[3], float lch[3])
```

Comment:
Better to put these in an anoymous name space rather than make them static.

Reply:
Fixed